src/Native/Utils.js

```diff
@@ -414,8 +415,9 @@ function toString(v)
 		var output = '';
 		for (var i in v)
 		{
-			if (i === 'ctor') continue;
-			var str = toString(v[i]);
+			var val = v[i];
+			if (val === v.ctor) continue;
```

Comment:
I'm looking into ways to get assets really small through other means, so I'm not sure this kind of stuff is gonna matter that much. Can we quantify how much advanced mode helps with size before getting into PRs to do that? Like, is it 20kb to 19kb? Is it making things faster? From there, we can assess better.